docs/fidesops/docs/guides/saas_connectors.md

```diff
@@ -0,0 +1,109 @@
+# How-To: Connect to SaaS Applications
+
+In this section we'll cover
+
+- What is a SaaS connection?
+- What is a SaaS config schema?
+- How do you configure a SaaS connection?
+
+## What is a SaaS connection?
+
+A SaaS (Software as a Service) connection is a connection type within Fidesops that allows a user to connect to a SaaS application (e.g., Mailchimp, Stripe, Slack, etc.) and execute data access and erasure requests against that application. These connections use functionality introduced in earlier sections (ConnectionConfigs, Datasets, Policies) but also introduce a new SaaS config specification to define how to connect to specific SaaS applications.
+
+## Supported SaaS applications
+
+The current implementation of the SaaS framework can support any SaaS application that uses these features:
+
+- Basic and bearer authentication
+- Data access via HTTP GET requests
+- Erasure via HTTP PUT requests
+
+The following features are planned for future releases and will allow for the configuration of broader types of connections:
+
+- OAuth 2.0 authentication
+- Pagination based on headers ands response contents
+- Retry logic based on status codes and response contents
+
+Full examples of a valid SaaS config and Dataset are currently available for Mailchimp.
+
+## What is a SaaS config schema?
+
+A SaaS connector is defined in two parts, the Dataset and the SaaS config. The Dataset describes the data that is available from a connector and the SaaS config describes how to connect and retrieve/update the data in a connector. If you contrast this to a database connector, the ways to retrieve/update data conform to a specification (such as SQL) and are consistent. When accessing data from APIs, each application or even different endpoints within the same application can follow different patterns. It was necessary to have a flexible configuration to be able to define the different access/update patterns.
+
+In short, you can think of the Dataset as the "what" (what data is available from this API) and the SaaS config as the "how" (how to access and erase the data).
+
+A SaaS config schema is broken into the following sections:
+
+- Metadata
+- Connector params
+- Client config
+- Test Request
```

Comment:
Nit: can we lowercase `request` to be in like with the items above?

Reply:
Nice catch